### DISCUSSION TITLE
JP Host

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#272] [Show and tell]
https://telegra.ph/DATA-12-23

Link originally found by @GiovanniSebastian

**Comment by what-a** [comment]
Working fine in the USA. I tested on both Windows and Linux.

**Comment by Central302** [comment]
Channel List 12-24
https://ghostbin.com/mPThL

**Comment by BellezaEmporium** [comment]
Updated file to reflect @Central302 's findings.

**Comment by makugafu** [comment]
Ever since the move to the hosts listed in the first link, I've been having issues. The biggest issue is that jp.m3u mostly does not match what is loaded by my media player. For example, I get Al Jazeera, some Turkish channels, and some Russian channels - these are definitely not listed in jp.m3u. The main culprits are the channels listed in the link. Those that aren't in the link are fine.

Trying on Android (via VLC, OTT Navigator and Perfect Player), Windows (via VLC) and LG smart tv (via SmartIPTV). All have the same issues. UK based.

Any ideas?

**Reply by BellezaEmporium** [reply]
It is due to regionalization of some channels. Some channels are basically streamed in the region Japan is in, hence why some channels outside of traditional Japanese channels are there.

**Reply by makugafu** [reply]
Makes sense somewhat, thank you. For clarification and my knowledge then, does the region _you_ are in determine what channel info the media player shows for a particular entry? Say, for exactly the same entry in jp.m3u, could what I see as Al Jazeera in the UK be Fuji in Japan?

Going further, if I were to create my own m3u from a version of jp.m3u prior to introducing the new hosts, could I probably get around the issues?

**Reply by BellezaEmporium** [reply]
For the first question, the channels being available in a whole region will be said as such. "MENA" as an example is Middle East and North Africa. If such channel gets to stream on all regions covering MENA, then we'll prefer saying it's streamed in MENA instead of telling all cities the program is being streamed in.

Of course, sometimes it gives weird things (North Korea getting access to Bloomberg as an example), but it is the way it is.

If you create your own version of the jp.m3u file, you won't see the regional streams in them. You'll only see the streams that were registered in Japan, that stream through Japan (or another country). We classify them by their country of origin. NHK World as an example is an international English stream, hence you'll see it on lots of playlists (if it's there of course)

**Comment by luongz** [comment]
It seem the port 8080 is died , but i tried port 8880 , it working (tested in Vietnam)

http://50.7.74.29:8880/hlsok/j00031/index.m3u8 (TBS)

![IMG_20220108_072853.jpg](https://user-images.githubusercontent.com/76716526/148623663-43228bdb-6b80-48cd-94e8-5ecad954cf5a.jpg)

**Reply by makugafu** [reply]
I can confirm this! Even from the UK!

Also tried some of the other problematic entries, changing the port as suggested. They also work.

**Reply by BellezaEmporium** [reply]
Thank you for the report. I'm changing them.
